blankshield.js

```diff
@@ -81,19 +83,22 @@
    * @param {Event} e The click event for a given anchor
    */
   function clickListener(e) {
-    var target, href, usedModifier, child;
+    var target, targetName, href, usedModifier, child;
 
     // Use global event object for IE8 and below to get target
     e = e || window.event;
-    target = e.target || e.srcElement;
+    // Won't work for IE8 and below for cases when e.srcElement
+    // refers not to the anchor, but to the element inside it e.g. an image
+    target = e.currentTarget || e.srcElement;
```

Comment:
``` javascript
// Won't work for IE8 and below for cases when e.srcElement
```
Is that comment regarding the old assignment `target = e.target || e.srcElement;` ? Should probably be deleted, no?